containers/signing-service/install-signing-service.sh.j2

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+# Install signing-service dependencies
+/tmp/install-dependencies.sh
```

Comment:
Please don't hard-code path to `/tmp/`. Use paths relative to script's location.

Reply:
Done.